integration/tls/tls_test.go

```diff
@@ -40,15 +40,18 @@ const (
 	prometheusConfigPath       = "config/prometheus.yml"
 	prometheusDockerImageName  = "prom/prometheus"
 	tlsRootCAPath              = "cert/RootCA.pem"
+	tlsCertificatePath         = "cert/ServerCertificate.crt"
+	tlsPrivateKeyPath          = "cert/ServerPrivateKey.key"
 	tlsServerCertPath          = "cert"
 	connectorDockerImageName   = "timestream-prometheus-connector-docker"
-	connectorDockerImagePath   = "../../resources/timestream-prometheus-connector-docker-image-1.0.0.tar.gz"
+	connectorDockerImagePath   = "../../resources/timestream-prometheus-connector-docker-image-1.0.3.tar.gz"
```

Comment:
Can we use `timestream.Version` here?

Reply:
Yes, I have revised to now use the imported version file.